duesseldorf.py

```diff
@@ -0,0 +1,163 @@
+import re
+from urllib.request import urlopen
+
+from bs4 import BeautifulSoup as parse
+
+from pyopenmensa.feed import OpenMensaCanteen
+from utils import Parser
+
+# this will match notes, e.g. "[R]" or "(1,10,12)"
+legend_regex = re.compile('(?:[\[(]{1}[ARFSVZ\d\,]+[\])])')
```

Comment:
What is the purpose of the non-capturing group? `{1}` also seems redundant as it's the default anyway.